project/src/main/kotlin/org/ionproject/core/class_section/ClassSectionController.kt

```diff
@@ -1,3 +1,32 @@
 package org.ionproject.core.class_section
 
-class ClassSectionController
\ No newline at end of file
+import org.ionproject.core.common.Action
+import org.ionproject.core.common.Siren
+import org.ionproject.core.common.SirenBuilder
+import org.ionproject.core.course_instance.CourseInstanceController
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class ClassSectionController(private val repo: ClassSectionRepo) {
+    companion object {
+        private val CLASS = arrayOf("class", "section")
+
+        fun buildHref(acr: String, term: String, id: String) =
+                "${CourseInstanceController.buildHref(acr, term)}/${id}"
+    }
+
+    @GetMapping("/v0/courses/{acr}/classes/{term}/{id}")
```

Comment:
You should add all the URI's to a separate file named "links", its easier to maintain